source/blog/2016-11-28-ember-2-10-released.md

```diff
@@ -0,0 +1,295 @@
+in the debug build before they are passed into helpers. As a result, you will
+get an error if you attempt to modify these objects directly. It is recommended
+that you clone these objects (e.g. using `Object.assign` and `Array#slice`)
+before making any modifications to them.
+
+(It has been brought to our attention that the last change might have an
+[unintended consequence](https://github.com/emberjs/ember.js/pull/14649).
+A fix is being worked on by [Matthew Beale](https://github.com/mixonic). Please
+refer to the linked issue for a detailed description and suggested workaround.)
+
+For more details on the changes in Ember.js 2.10, please review the
+[Ember.js 2.10.0 release page](https://github.com/emberjs/ember.js/releases/tag/v2.10.0).
+
+### Upcoming changes in Ember.js 2.11
+
+Currently, no new features are planned for 2.11. However, there are a few
+notable changes:
+
+- Similar to the change to helper arguments, concatenated properties (such as
+  `classNames` and `classNameBindings`) are now [frozen](https://github.com/emberjs/ember.js/pull/14389)
+  in debug builds to help track down unintended mutations.
+
+- The legacy `render` helper (i.e. `{{render 'foo'}}`) has been [deprecated](https://github.com/emberjs/ember.js/pull/14441).
+
+- The private `Component#renderToElement` API has also been [deprecated](https://github.com/emberjs/ember.js/pull/14482).
+
+For more details on the upcoming changes in Ember.js 2.11, please review the
+[Ember.js 2.11.0-beta.1 release page](https://github.com/emberjs/ember.js/releases/tag/v2.11.0-beta.1).
+
+---
+
+## Ember Data
+
+Ember Data is the offical data persistence library for Ember.js applications.
+
+### Changes in Ember Data 2.10
+
+Ember Data 2.10 represents the work of more than 10 direct contributors with over
+20 commits.
+
+There have been no user facing changes in Ember Data 2.10. Under the
+hood, [@runspired](https://github.com/runspired) has instrumented
+Ember Data with some internal tools to make it easier to track the
+performance of Ember Data. This instrumentation is stripped from the
+production build of Ember Data and should have no impact on existing
+applications that rely on Ember Data.
+
+[@runspired](https://github.com/runspired) also
+[improved the messaging](https://github.com/emberjs/data/pull/4578)
+around assertions and warnings when an adapter responds with no `id`
+for a record.
+
+For more details on the changes in Ember Data 2.10, please review the
+[Ember Data 2.10.0 release page](https://github.com/emberjs/data/releases/tag/v2.10.0).
+
+### Upcoming changes in Ember Data 2.11
+
+Ember Data 2.11 is already shaping up to be a big release with many
+performance and documentation improvements.
+
+#### HasMany Computed Property Keys
+
+One of the performance improvements included in the Ember Data
+2.11.0-beta.1 release fixes an issue where Ember Data was doing too
```

Comment:
I think this should be "2.11" per https://github.com/emberjs/website/pull/2727/commits/fa1d1d5ea6ba05240660890ec1a2c8f82a243455